plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/AbstractPinotIntegrationSmokeTest.java

```diff
@@ -121,13 +122,18 @@
 
     protected abstract boolean isSecured();
 
+    protected String getPinotImageName()
+    {
+        return PINOT_LATEST_IMAGE_NAME;
```

Comment:
In other connectors the default is the oldest version. I think it makes more sense since generally version updates add things so we should have wider coverage for older versions.

Reply:
Sounds good, I also renamed `TestPinotWithoutAuthenticationIntegrationSmokeTestPreviousVersion` to `TestPinotWithoutAuthenticationIntegrationSmokeTestLatestVersion`.